bin/install

```diff
@@ -156,8 +158,23 @@ get_platform() {
   get_platform_${g_platform_pattern}
 }
 
+get_cpu() {
+  local machine_hardware_name
+  machine_hardware_name="$(uname -m)"
+
+  case "$machine_hardware_name" in
+    'x86_64') local cpu_type="amd64";;
```

Comment:
does not work on my linux env.

It tries to download <https://github.com/derailed/k9s/releases/download/v0.24.14/k9s_Linux_amd64.tar.gz>
when it should be <https://github.com/derailed/k9s/releases/download/v0.24.14/k9s_Linux_x86_64.tar.gz>

Reply:
fixed by your latest commit